src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -431,17 +425,16 @@ class OpenGLDrawingEngine final : public IDrawingEngine
                     dst += pitch;
                 }
             }
-            delete[] _bits;
         }
 
-        _bits = newBits;
+        _bits.reset(newBits);
```

Comment:
I feel like `newBits` should also be an `unique_ptr<uint8_t[]>` that is moved into `_bits` instead.